tests/io/test.xpi.js

```diff
@@ -60,25 +60,26 @@ describe('Xpi.open()', function xpiCallback() {
     };
   });
 
-  it('should resolve with zipfile', () => {
+  it('should resolve with zipfile', async () => {
     const myXpi = new Xpi('foo/bar', this.fakeZipLib);
     // Return the fake zip to the open callback.
     this.openStub.yieldsAsync(null, this.fakeZipFile);
-    return myXpi.open()
-      .then((zipfile) => {
-        expect(zipfile.testprop).toEqual('I am the fake zip');
-      });
+
+    const zipfile = await myXpi.open();
+    expect(zipfile.testprop).toEqual('I am the fake zip');
   });
 
-  it('should reject on error', () => {
+  it('should reject on error', async () => {
     const myXpi = new Xpi('foo/bar', this.fakeZipLib);
     // Return the fake zip to the open callback.
     this.openStub.yieldsAsync(new Error('open() test error'));
-    return myXpi.open()
-      .then(unexpectedSuccess)
-      .catch((err) => {
-        expect(err.message).toContain('open() test');
-      });
+
+    try {
```

Comment:
As for #1733, we can use `await expect(...).rejects.toThrow("...")` here (and for the other `try/catch` from this test file).

Reply:
@rpl I've changed some Jest methods in this file